includes/class-branding.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * The Class for adding branding to the dashboard.
+ *
+ * @package aspire-update
+ */
+
+namespace AspireUpdate;
+
+/**
+ * The Class for adding branding to the dashboard.
+ */
+class Branding {
+	/**
+	 * Hold a single instance of the class.
+	 *
+	 * @var object
+	 */
+	private static $instance = null;
+
+	/**
+	 * The Constructor.
+	 */
+	public function __construct() {
+		$admin_settings = Admin_Settings::get_instance();
+		if ( $admin_settings->get_setting( 'enable', false ) ) {
+			add_action( 'admin_notices', [ $this, 'output_admin_notice' ] );
+			add_action( 'admin_enqueue_scripts', [ $this, 'admin_enqueue_scripts' ] );
+		}
+	}
+
+	/**
+	 * Initialize Class.
+	 *
+	 * @return object
+	 */
+	public static function get_instance() {
+		if ( null === self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Enqueue scripts and styles.
+	 *
+	 * @param string $hook The page identifier.
+	 * @return void
+	 */
+	public function admin_enqueue_scripts( $hook ) {
+		if ( defined( 'AP_REMOVE_UI' ) && AP_REMOVE_UI ) {
+			return;
+		}
+
+		$allowed_screens = [
+			'update-core.php',
+			'plugins.php',
+			'plugin-install.php',
+			'themes.php',
+			'theme-install.php',
+		];
+
+		if ( in_array( $hook, $allowed_screens, true ) ) {
+			wp_enqueue_style( 'aspire_update_settings_css', plugin_dir_url( __DIR__ ) . 'assets/css/aspire-update.css', [], AP_VERSION );
+		}
+	}
+
+	/**
+	 * Output admin notice.
+	 *
+	 * @return void
+	 */
+	public function output_admin_notice() {
+		if ( defined( 'AP_REMOVE_UI' ) && AP_REMOVE_UI ) {
+			return;
+		}
+
+		$current_screen = get_current_screen();
+		if ( ! $current_screen instanceof \WP_Screen ) {
+			return;
+		}
+
+		$message = '';
+		switch ( $current_screen->base ) {
+			case 'plugins':
+			case 'plugin-install':
+				$message = sprintf(
+					/* translators: 1: The name of the plugin, 2: The documentation URL. */
+					__( 'Your plugin updates are now powered by <strong>%1$s</strong>. <a href="%2$s">Learn more</a>', 'AspireUpdate' ),
+					'AspireUpdate',
+					__( 'https://docs.aspirepress.org/aspireupdate/', 'AspireUpdate' )
```

Comment:
Do we need to translate the URLs?

Reply:
I don't think so.

Reply:
If the contents of the URL are translated (or will be translated), then we should translate the URL. Otherwise not.

I think translating the docs would show that we recognise a significant portion of sites are non-English, and translating the URL means we're prioritizing the user's choice of language, not the project's.

@asirota Are there plans to make the docs site multilingual with a GitHub Pages plugin and have our polyglots provide the translations? (I can help with the setup if help is needed)

If so, then keeping the URL translated in this PR could save us time doing it later.

Reply:
I see no harm in having it, but I don't think we are jumping into translating docs soon as things are changing almost daily right now and translators would have a hard time catching up. Probably good to take up once we are more stable.

Reply:
When creating a new PO file the urls follow when extracting strings, so better to keep them and its also prepared for future if docs/website will be in other languages.

Reply:
Translating docs into multiple languages should happen in concordance of our official supported languages. It's a future project not yet scoped or planned. Let's keep urls as is for now.